back/app/policies/event_file_policy.rb

```diff
@@ -1,33 +1,25 @@
 # frozen_string_literal: true
 
 class EventFilePolicy < ApplicationPolicy
-  class Scope
-    attr_reader :user, :scope
-
-    def initialize(user, scope)
-      @user  = user
-      @scope = scope
-    end
-
+  class Scope < ApplicationPolicy::Scope
     def resolve
-      event_ids = Pundit.policy_scope(user, Event).ids
-      scope.where(event_id: event_ids)
+      scope.where(event: scope_for(Event))
     end
   end
 
   def create?
-    EventPolicy.new(user, record.event).update?
+    policy_for(record.event).update?
```

Comment:
Reading this, thinking it could be an opportunity to rename `policy_for` to `policy`? This way it's the same in policies and controllers, will help with making sure it's easy to remember and keep on applying it consistently.

Reply:
Yes, I considered that, but in the end I preferred keeping consistency between `ApplicationPolicy#policy_for` and `ApplicationPolicy#scope_for`. We can't use `scope` instead of `scope_for` because it conflicts with the `attr_reader` in `ApplicationPolicy::Scope` (and, anecdotically, with a `scope` helper in `ApplicationPolicy` that probably nobody uses or is aware of—including me bf this pr).